.travis.yml

```diff
@@ -6,70 +6,78 @@ addons:
 
 env:
   global:
-    - SAUCE_USERNAME=snay
```

Comment:
The Travis part should be a separate PR

Reply:
yep, fucked up merged. will revert and rebase now.